### ISSUE TITLE
Problem running the demos

### ISSUE THREAD

**Issue by berserk82** [kripken/ammo.js#170]
Hello there.
My name is Ivano Arrighetta and I'm Italian.
I tried opening the demos in both Firefox and Chrome, the latter with a webserver.
The only demo I was able to "see" was webgl_demo, but I can just see the plane.
No cubes, no physics.
Does ammo.js require something additional rather than just a web browser? (eg. node or something)

Thanks in advance for any help.

Bye, ivano.

**Comment by xprogram** [comment]
Yeah, I have just experienced this myself. I tried the links on the README and the "loading..." screen just showed forever. I opened up the development console and found that the browser is reporting that 'Ammo.btTransform' is not a constructor. I did some testing and found that 'Ammo' is a function that returns an object. So now every instance of `Ammo` should be `Ammo()`?

**Comment by berserk82** [comment]
The code should be tested before a commit...

**Comment by kripken** [comment]
We did change the API to be async, so `Ammo()` is needed. Perhaps not all the demos were updated for that, we should fix it.

**Comment by berserk82** [comment]
When?

**Comment by kripken** [comment]
As soon as someone finds time, fixing this does not depend on anything.

(Hopefully the docs are good enough to explain what needs to be done, but if not, if someone tries and gets stuck, feel free to ask me stuff, of course.)

**Comment by derivou** [comment]
hello all, not sure I'm on the right place for my related observation : I'm using ammo.js (taken from  three.js distrib r87). I collide two softbodies (simple spheres - rolling on a solid ground)  and the collision ends with a kind of explosion of one of the softbodies ... see hereafter  . I used different pressure ans mass but the result identical. 
If using the ammo.js from github I get "mmo.btTransform is not a constructor". 
Is there any hint to solve this ? 
![ammo](https://user-images.githubusercontent.com/4748407/33023446-7ab42910-ce08-11e7-9ada-6ae25b13455d.jpg)

**Comment by derivou** [comment]
I modified the pressure of softbodies in the exemple (softbody volume) given in the ammo.js distribution. With p=1000 the two objects seem to attract each other and finally explode as in my own exemple. It seems this is part of the normal behavior but what does it mean ?

**Comment by derivou** [comment]
Is there a demo mixing ammo physics and skeletal animation (material.skinned = true) ?

**Comment by willeastcott** [comment]
I had a go at this [here](https://playcanvas.com/project/431888/overview/ragdoll). Needs a bit more work though.

**Comment by derivou** [comment]
Oh thanks for confirmation ; that's nice.  I use ammo.js with three.js and the object stays there (I can see the collision effect on others) but I'm afraid there is something with my code in Three.js because skin is not rendered as soon as I set material.skinning=true. OK ... have to dig more.

**Comment by willeastcott** [comment]
Ahhh, can't help with Three I'm afraid. Maybe someone else here knows it. Good luck!

**Comment by derivou** [comment]
Thanks however, at least I know ammo.js is not the problem. So, I have a direction to work.
